web/packages/teleterm/build_resources/linux/after-install.tpl

```diff
@@ -24,6 +24,36 @@ if hash update-desktop-database 2>/dev/null; then
   update-desktop-database /usr/share/applications || true
 fi
 
+# Install apparmor profile. (Ubuntu 24+)
+# First check if the version of AppArmor running on the device supports our profile.
+# This is in order to keep backwards compatibility with Ubuntu 22.04 which does not support abi/4.0.
+# In that case, we just skip installing the profile since the app runs fine without it on 22.04.
+#
+# Those apparmor_parser flags are akin to performing a dry run of loading a profile.
+# https://wiki.debian.org/AppArmor/HowToUse#Dumping_profiles
+#
+# Unfortunately, at the moment AppArmor doesn't have a good story for backwards compatibility.
+# https://askubuntu.com/questions/1517272/writing-a-backwards-compatible-apparmor-profile
+if apparmor_status --enabled > /dev/null 2>&1; then
+  APPARMOR_PROFILE_SOURCE='/opt/${sanitizedProductName}/resources/apparmor-profile'
```

Comment:
Does electron-builder include this file now somehow?

Reply:
Yes. 
https://github.com/electron-userland/electron-builder/blob/bff46ec41c4a7715cc06f7dfd6ff95f8e4bbe869/packages/app-builder-lib/templates/linux/apparmor-profile.tpl

I verified that it's properly copied to `etc/apparmor.d`.